rpmlb/builder/base.py

```diff
@@ -80,7 +80,13 @@ def run(self, work, **kwargs):
                     continue
 
                 self.prepare(package_dict, **kwargs)
-                self.build_with_retrying(package_dict, **kwargs)
+
+                # Repeat the call in case of failure
+                retry_call(
+                    self.build, fargs=(package_dict,), fkwargs=kwargs,
+                    tries=kwargs.get('retry', 0)+1,  # 0 tries == no call
```

Comment:
I like below kind of way, because of 1 line 1 statement rule.

```
retry_num = kwargs.get('retry', 0)
retry_call(
    self.build, fargs=(package_dict,), fkwargs=kwargs,
    tries=retry_num+1
...
```